src/gerbil/runtime/gambit.ss

```diff
@@ -0,0 +1,65 @@
+;;; -*- Gerbil -*-
+;;; © vyzo
+;;; Gambit builtins
+prelude: :gerbil/core
+package: gerbil/runtime
+namespace: #f
+
+(export #t)
+
+;; compatibilityf for prerelease gambit changes
+(cond-expand
```

Comment:
compatibility

Reply:
I will fix in follow up, i dont want to wait another hour for CI.